app/models/match_decisions/eight/confirm_match_success_dnd_staff.rb

```diff
@@ -0,0 +1,92 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/boston-cas/blob/production/LICENSE.md
+###
+
+module MatchDecisions::Eight
+  class ConfirmMatchSuccessDndStaff < ::MatchDecisions::Base
+    # validate :note_present_if_status_rejected
```

Comment:
Is this supposed to be uncommented?

Reply:
This is commented out but left present in a a whole lot of decisions... A clean up pass is a good idea.